packages/commons/core-utils/jest.config.cjs

```diff
@@ -1,7 +1,7 @@
 /** @type {import('ts-jest').JestConfigWithTsJest} **/
 module.exports = {
   testEnvironment: "node",
-  testMatch: [ "**/__tests__/**/*.ts?(x)", "**/?(*.)+(spec|test).ts?(x)" ],
+  testMatch: [ "**/?(*.)+(spec|test).ts?(x)" ],
```

Comment:
Context: The unit test was failing because jest was trying to run the test's `*.d.ts` file from the compiled output in `lib`. I saw that `cli/workspace-runner` was using vitest and the vitest config was `include: ["**/*.{test,spec}.ts"]`, which obv doesn't include `.d.ts` files. 

Thought about using the same setup with vitest here in core-utils, but didn't want to add a dependency and then update pnpm-lock.yaml for something this minor.